modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-service/src/main/java/com/liferay/portal/reports/engine/console/service/impl/EntryLocalServiceImpl.java

```diff
@@ -123,7 +123,7 @@ public Entry addEntry(
 		entry.setCreateDate(serviceContext.getCreateDate(date));
 		entry.setModifiedDate(serviceContext.getModifiedDate(date));
 		entry.setDefinitionId(definitionId);
-		entry.setFormat(format);
+		entry.setFormat(reportFormat.toString());
```

Comment:
@dantewang @JanisZhang This changes what stored into database which needs an upgrade for existed data.